tests/calculator-homework.spec.js

```diff
@@ -0,0 +1,106 @@
+const { test, expect } = require('@playwright/test');
+const { CalculatorPage } = require('../pages/calculatorPage')
+
+const calculationActions = ['0', '1', '2', '3', '4'];
+const calculationActionName = ['Add', 'Subtract', 'Multiply', 'Divide', 'Concatenate'];
+const integerCalculationResults = ['20', '0', '100', '1', '1010'];
+const calculationResults2 = ['5', '0', '6.25', '1', '2.52.5'];
+const integerOnlyResults = ['5', '0', '6', '1'];
+const versions = ['1', '2', '3', '4', '5', '6', '7', '8', '9'];
+let answer, isVisible, isEnabled;
+
+
+versions.forEach(version => {
+  test.describe('', () => {
+    let page;
+    test.beforeAll(async ({ browser }) => {
+      page = await browser.newPage();
+      startPage = new CalculatorPage(page);
+      
+    });
+    test.beforeEach(async () => {
+      await startPage.goto();
+      await page.selectOption('#selectBuild', version);
+    });
+
+
+    calculationActions.forEach(calculationAction => {
+      test.only(`Test if ${calculationActionName[calculationAction]} is working with integers on version ${version}`, async () => {
+        await startPage.performCalculation('10', '10', calculationAction);
+        answer = await page.inputValue('#numberAnswerField');
+        expect(answer).toBe(integerCalculationResults[calculationAction]);
+      });
+    });
+
+
+    calculationActions.forEach(calculationAction => {
+      test.only(`Test if ${calculationActionName[calculationAction]} is working with non integers on version ${version}`, async () => {
+        await startPage.performCalculation('2.5', '2.5', calculationAction);
+        answer = await page.inputValue('#numberAnswerField');
+        expect(answer).toBe(calculationResults2[calculationAction]);
+      });
+    });
+
+
+    for (let i = 0; i < 4; i++) {
+      test.only(`Test if integer only option is enabled, visible and working during ${calculationActionName[i]} on version ${version}`, async () => {
+        await startPage.performCalculation('2.6', '2.5', calculationActions[i]);
+        await page.check('#integerSelect');
+        isEnabled = await page.isEnabled('#integerSelect');
+        isVisible = await page.isVisible('#integerSelect');
+        answer = await page.inputValue('#numberAnswerField');
+        expect(isEnabled).toBe(true);
```

Comment:
isEnabled, isVisible check'ai cia nereikalingi, nes paskutinis is esmes pagaus jeigu jis dings ar taps disabled